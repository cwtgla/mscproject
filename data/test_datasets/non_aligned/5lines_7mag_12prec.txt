127.4095
69.3812
0.1121
-88.2294
-127.0991