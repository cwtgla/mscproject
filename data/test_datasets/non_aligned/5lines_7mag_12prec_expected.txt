127
255
244
94
228
0
70
29
136
246
255
61
240